packages/multi-select-combo-box/src/vaadin-multi-select-combo-box-container.js

```diff
@@ -27,6 +27,7 @@ class MultiSelectComboBoxContainer extends InputContainer {
           display: flex;
           width: 100%;
           min-width: 0;
+          gap: var(--_wrapper-gap);
```

Comment:
Added reset for this custom CSS property to Lumo as otherwise it breaks visual tests.
Moving this CSS to `styles` folder is also an option but that sounds like an overkill.